cypress/integration/visitorCanSeeListOfArticles.spec.js

```diff
@@ -9,11 +9,6 @@ describe("Visitor can see articles on the homepage", () => {
     cy.visit("http://localhost:3001");
   });
 
-  it("Two articles are displayed", async () => {
-    cy.get("#article_1");
-    cy.get("#article_2");
-  });
-
```

Comment:
change below or change fixture file for title and body

Reply:
fixed